pkg/costmodel/allocation_helpers.go

```diff
@@ -655,6 +697,7 @@ func applyGPUsAllocated(podMap map[podKey]*pod, resGPUsRequested []*prom.QueryRe
 
 			hrs := thisPod.Allocations[container].Minutes() / 60.0
 			thisPod.Allocations[container].GPUHours = res.Values[0].Value * hrs
+			thisPod.Allocations[container].GPURequestAverage = res.Values[0].Value
```

Comment:
Just confirming -- is this always true? That request == allocation? I.e. you cannot somehow use more than you request, and in order to use, you must request?

Reply:
For now, yes this is always true! When you request to use a GPU you need to request the full GPU. In yaml it looks like this:

```yaml
      # https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/
      resources:
        limits:
          nvidia.com/gpu: 1
```

Therefore the max(usage, request) will always equal the request. I'll leave a comment clarifying that idea. In the future we'd like to support [GPU time slicing](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-sharing.html) in which case this would no longer be true.